cts/cts-cli.in

```diff
@@ -1153,6 +1153,10 @@ class DatesRegressionTest(RegressionTest):
                  "iso8601 -d '2009-01-31 00:00:00Z' -D P2M -E '2009-03-31 00:00:00Z'"),
             Test("2009-01-31 + 3 Months",
                  "iso8601 -d '2009-01-31 00:00:00Z' -D P3M -E '2009-04-30 00:00:00Z'"),
+            Test("Test correct immediate plus-sign at timezone parsing",
+                 "iso8601 -d '2019-04-25T21:42:54+0200' -S"),
+            Test("Like the previous, with exotic timezone to avoid false negative",
```

Comment:
Not sure why `"+0420"` would be considered "exotic" or expected to perhaps cause a problem. (I know this goes all the way back to poki's original PR.)